manifests/repo.pp

```diff
@@ -119,6 +119,14 @@
         Apt::Source['zabbix'] -> Package<|tag == 'zabbix'|>
         Class['Apt::Update']  -> Package<|tag == 'zabbix'|>
       }
+      'Suse': {
+        $minorrelease = $facts[os][release][minor]
+        exec { 'zypper addrepo':
```

Comment:
I'm not very familiar with susel, can we use another resource and not exec here? For example https://github.com/voxpupuli/puppet-zypprepo ?